From c39f16102ed1b22ad796e2416bb60792b9aa5571 Mon Sep 17 00:00:00 2001
From: Andrei Fadeev <andrei@webcontrol.ru>
Date: Wed, 15 Mar 2017 09:13:34 +0300
Subject: [PATCH] Fix build on MSVC

The inline keyword is available only for C++ in MSVC.
So we need to use Microsoft specific __inline.
---
 meson.build | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/meson.build b/meson.build
index a513f5f..7d9afd9 100644
--- a/meson.build
+++ b/meson.build
@@ -159,6 +159,14 @@ if libtype == 'shared'
   endif
 endif
 
+# The inline keyword is available only for C++ in MSVC.
+# So we need to use Microsoft specific __inline.
+if host_system == 'windows'
+  if cc.get_id() == 'msvc'
+    conf.set('inline', '__inline')
+  endif
+endif
+
 # Dependencies
 dl_dep = cc.find_library('dl', required: false)
 gl_dep = dependency('gl', required: false)
-- 
2.7.4

